packages/appium/lib/appium.js

```diff
@@ -216,6 +216,12 @@ class AppiumDriver extends BaseDriver {
 
       // This assignment is required for correct web sockets functionality inside the driver
       driverInstance.server = this.server;
+
+      // Drivers/plugins might also want to know where they are hosted
+      driverInstance.serverHost = this.args.address;
```

Comment:
maybe we should take this opportunity to add a setter or setters to the driver class for this stuff.  at the very least it'd be less of a mystery where these properties originate

Reply:
https://github.com/appium/appium/issues/16156